coalib/settings/Section.py

```diff
@@ -187,6 +187,9 @@ def add_or_create_setting(self,
         if self.__contains__(key, ignore_defaults=True) and allow_appending:
             val = self[key]
             val.value = str(val._value) + '\n' + setting._value
+            if(key == 'ignore'):
```

Comment:
no.  we are not having special parsing and string manipulation of only the `ignore` Setting, and we're definitely not putting post-processing string manipulation of Setting's into the Section module .  Nobody will be able to find it here, and they will be very confused.

Reply:
You say that I suggested something previously, but I cant find which comment you are referring to.  Every comment has a unique link.  Please give me a URL to the comment you are referring to.

If an unrelated test fails, you need to understand that test, and what is achieves.

In the case of `test_section_inheritance` , it loads `tests/settings/section_manager_test_files/inherit_coafile` , which has a `ignore +=` without any previous value.

Then it dynamically changes the value of the magic `cli` section setting

`sections['cli']['ignore'] = './user'`

And it expects that the two sections magically convert into a comma separated list.

This is why you can not - must not - do post processing string manipulation.

The ConfParser must give to Setting the literal value of setting in that section.  Then Setting manages the value after that.